deltachat-jsonrpc/src/api/types/chat.rs

```diff
@@ -61,6 +61,13 @@ pub struct FullChat {
     is_contact_request: bool,
 
     is_device_chat: bool,
+    /// Note that this is different from
+    /// [`ChatListItem::is_self_in_group`](`crate::api::types::chat_list::ChatListItemFetchResult::ChatListItem::is_self_in_group`).
+    /// This property should only be accessed
+    /// when [`FullChat::chat_type`] is [`Chattype::Group`].
+    //
+    // We could utilize [`Chat::is_self_in_chat`],
```

Comment:
This is implementation detail. I see that it is not a documentation comment, but I'd even completely remove it, you anyway need to look at the implementation to see if there is an additional query to fill this field or not.